lib/gen/context_menus.dart

```diff
@@ -7,8 +7,8 @@
  */
 library chrome.contextMenus;
 
-//import 'context_menus_internal.dart';
```

Comment:
I was searching through the diff and noticed this, were you previously hand-removing the internal reference here?